tests/agent_unittests/test_agent_protocol.py

```diff
@@ -196,7 +196,7 @@ def test_send(status_code):
 
     assert request.headers == {
         "Content-Type": "application/json",
-        "custom-header": u"value",
+        "custom-header": "value",
```

Comment:
It looks like the u was accidentally dropped here.
```suggestion
        "custom-header": u"value",
```